bool useSensors
---
float32[] positions